whisper/stt/__init__.py

```diff
@@ -12,6 +12,16 @@
 # see https://github.com/guillaumekln/faster-whisper/issues/150
 os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"  # GPU in the right order
 
+if os.environ.get("VAD","auditok") in [True, "true", 1]:
```

Comment:
You will never goot a boolean or an integer with os.environ.
It should be `os.environ.get("VAD","auditok").lower() in ["true", "1"]`
(same for False).

Also you should check here that the specified VAD method is correct.
To avoid only throwing at the first call.
(if the configuration is not right, we have to fail early)